schemas/constants.js

```diff
@@ -84,4 +84,8 @@ module.exports = {
 
   // Slugid pattern, for when-ever that is useful
   "slugid-pattern":  "^[a-zA-Z0-9-_]{22}$",
+
+  // Pattern for base64 encoded strings
+  "base64-pattern":  "^(?:[A-Za-z0-9+/]{4})*" +
+                     "(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)$"
 };
```

Comment:
Interesting, I didn't realise base64 strings were always a multiple of 4 characters. Maybe worth adding a comment to explain that they are a multiple of 4 characters, with either one or two = characters as RHS padding.